dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ResourcesServiceImpl.java

```diff
@@ -182,10 +165,7 @@ private String getFullName(String currentDir, String name) {
      */
     @Override
     @Transactional
-    public Result<Object> createResource(User loginUser,
-                                         String name,
-                                         ResourceType type,
-                                         MultipartFile file,
+    public Result<Object> uploadResource(User loginUser, String name, ResourceType type, MultipartFile file,
```

Comment:
I am not sure how the user can only modify the resource file under resource path. Could you please give some brief description